.tool-versions

```diff
@@ -0,0 +1 @@
+java corretto-21.0.3.9.1
```

Comment:
Would we want the [lambda runtime](https://github.com/guardian/anghammarad/blob/909c98b04aeb24675d45cef795307d666e1eb81f/cloudformation/cfn.yaml#L64) to match this version?

Reply:
Yeah, good point! Done with 5a95383460e96769032675e42be189bb8a0352a4.